### DISCUSSION TITLE
General: Radioactive sources

### DISCUSSION THREAD

**Discussion by Captainslarty** [Gissio/radpro#76] [General]
Hi all, regarding radioactive sources. Of course, the simplest is the smoke detector Americium elements, available for about 5 dollars / euros on aliexpress. Then there are the excellent radium painted compass units - ex military - still available from Ukraine. Also, the Thoriated lamp mantles and also welding rods. (all from ebay)

Anyone got other good sources ?

Joe

**Comment by alfmck** [comment]
For beta testing I use potassium fertilizer bought in garden shop. But for measuring on GC-01 with J321 i have to remove back cover and place a tube on the sample. The Average results are 105.5 cpm/min, accuracy 4.3% while environment shows 23 cpm.
I;m planning to cut a window on the back cover and cover the hole with photoblastic tape used in some cables. 
FS-5000 has some holes on the back cover but shows 30-40 cpm. After removing back cover and placing tube on the sample it shows 130-150 cpm.

**Reply by Captainslarty** [reply]
Hi sir, beta 'should' pass through the plastic, it normally takes about 3 to 4 mm of aluminum to shield beta, however, yes, definitely to cutting a windows. It sounds like the source is mainly ALPHA ? /  for beta, Kapton tape or similar would be fine. Do you know what isotope you are actually measuring please ?
Joe

**Reply by alfmck** [reply]
I guess that the main reason is not a type of plastic, but the distance between the tube and sample. It is about 4mm for FS-5000.
But my friend drilled some holes in the back cover of his GC-01 and got twice increasing of indication on that sample.

**Reply by Captainslarty** [reply]
What is the reading in uS/h ?

(Aslo, gave just ordered a source of Strontium 90 (it will be in it's second half life be now)  with a Ukrainian DP-V5 meter. That is pretty damn hot.)  - I will add it to my thorium, radium and amricumium.
